### PR TITLE
Npm packaging for custom electron downloader

### PR DESCRIPTION
**Pull request by whutchinson98**

### CODE REVIEW

File: .github/workflows/assign-author.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  pull_request:
+    types: [opened, reopened]
+
+jobs:
+  assignAuthor:
+    name: Assign author to PR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Assign author to PR
+        uses: technote-space/assign-author@v1
```
Comment (chase) [inline .github/workflows/assign-author.yml:13]: This seems out of scope, why is this necessary? I want to keep the amount of untrusted third-party action (basically not from GitHub/Microsoft/etc.) limited. It's a bad idea to run arbitrary actions because this is a critical supply chain asset.

File: ci/updatePackageForRelease.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+const NPM_PACKAGE_PATH = `${__dirname}/../src/electron/npm`;
+const PACKAGE_JSON_PATH = `${NPM_PACKAGE_PATH}/package.json`;
+const ELECTRON_D_TS_PATH = `${NPM_PACKAGE_PATH}/electron.d.ts`;
+
+module.exports = async ({github}) => {
+  const {VERSION} = process.env;
+  // Get the release by version. This will give us the asset ids
+
+  // TODO: Download electron.d.ts from release and move into npm
```
Comment (chase) [inline ci/updatePackageForRelease.js:11]: It looks like we do this on lines 26-34 below, if that's the case, we can probably remove the TODO

**Review by chase — CHANGES_REQUESTED** [review]

File: .github/workflows/assign-author.yml (hunk 1 above)
Comment (whutchinson98) [reply .github/workflows/assign-author.yml]: Removed

File: ci/updatePackageForRelease.js (hunk 2 above)
Comment (whutchinson98) [reply ci/updatePackageForRelease.js]: removed todo line

**Review by chase — APPROVED** [review]

**Review by chase — APPROVED** [review]
